doc/release-notes.md

```diff
@@ -53,6 +53,27 @@ the Linux kernel, macOS 10.8+, and Windows Vista and later. Windows XP is not su
 Bitcoin Core should also work on most other Unix-like systems but is not
 frequently tested on them.
 
+Notes for 0.15.0
+================
+
+Current SegWit Support
```

Comment:
nit s/Support/support

Reply:
Fixed, although title capitalisation is a bit inconsistent already in the file